lib/ansible/cli/_ssh_askpass.py

```diff
@@ -0,0 +1,25 @@
+# Copyright: Contributors to the Ansible project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import annotations
+
+import os
+import sys
+import typing as t
+from multiprocessing.shared_memory import SharedMemory
+
+
+def main() -> t.NoReturn:
+    kwargs: dict[str, t.Any] = {}
```

Comment:
Looks like this doesn't really need any:
```suggestion
    kwargs: dict[str, bool] = {}
```
Though, MyPy will probably demand
```suggestion
    kwargs: dict[str, bool] | dict[t.Never, t.Never] = {}
```
*(it's probably better to use a typed dict, still)*